workflow/tests/test_models.py

```diff
@@ -3,8 +3,12 @@
 from django.test import TestCase, override_settings, tag
 
 import factories
-from workflow.models import TolaUser, Office
+from workflow.models import TolaUser, Office, Group
 
+ROLE_ORGANIZATION_ADMIN = 'OrgAdmin'
```

Comment:
You don't need to define all roles if you are not going to use them. Secondly, they are already here defined, so you can import the ones you need: https://github.com/yilmazbekdemir/TolaActivity/blob/4905fccb07bcc98e25df3914b8f2340d564e3ca6/workflow/models.py#L22